public/404.html

```diff
@@ -1,67 +1,103 @@
 <!DOCTYPE html>
-<html>
+<html lang='en'>
 <head>
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  body {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+<title>RubyBench - Long Running Ruby Benchmark</title>
+<meta content='width=device-width, initial-scale=1' name='viewport'>
+<!-- Loads Freight Sans Pro font with these font weights: 300, 400, 700 -->
 
-  div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
+<link rel="stylesheet" media="screen" href="https://fonts.googleapis.com/css?family=Montserrat:400,700" data-turbolinks-track="reload" />
+<link rel="stylesheet" media="all" href="/assets/application.self-745519abfad0a8dbde6e0cd8fe1492f9d36e30a9f06a1a1785b8c57805501903.css?body=1" data-turbolinks-track="reload" />
```

Comment:
What about the compiled stylesheet reference? Do you think we can get that working on a static page? Can you point me to some docs that will help?